### ISSUE TITLE
Compose does not work with automatic accessibility testing

### ISSUE THREAD

**Issue by DavidCorrado** [android/android-test#2240]
### Description
View system has automatic ADA testing via https://developer.android.com/training/testing/espresso/accessibility-checking

As far as I am aware this does not work in compose.  Is there ways to get this to work?  Is there a feature request for adding this?
